src/clr-angular/utils/smart-popover/providers/smart-popover-toggle.service.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
```

Comment:
Noticed many of the copyrights look like they are formatted incorrectly, can you check them all?

Reply:
There was a space in my copyright generator. :(
Will fix.